assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -416,6 +416,13 @@ che.infra.kubernetes.ingress.annotations_json=NULL
 che.infra.kubernetes.pod.security_context.run_as_user=NULL
 che.infra.kubernetes.pod.security_context.fs_group=NULL
 
+# Defines grace termination period for pods that will be created by Kubernetes / OpenShift infrastructures
+#
+# Grace termination period of Kubernetes / OpenShift pods defaults to 30 seconds, which results in
```

Comment:
Doc says `pods defaults to 30 seconds` but default value is
```
che.infra.kubernetes.pod.termination_grace_period_sec=0
```
Please check and sync these values

Reply:
+1

Reply:
well, default to `0` is done deliberately and I think we should stick to it  (which would significantly decrease time required for workspace stop on k8s / openshift). Probably I should just remove info about default k8s config, which you find misleading